substrate/frame/system/src/lib.rs

```diff
@@ -347,6 +348,9 @@ pub mod pallet {
 			/// Weight information for the extrinsics of this pallet.
 			type SystemWeightInfo = ();
 
+			/// Weight information for the extensions of this pallet.
```

Comment:
This line of docs would be more helpful down below in the `trait Config`.

Reply:
Done.